### PR TITLE
Fix format ‘verb’ in logs

### PR DESCRIPTION
**Pull request by houjun41544**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/volume/attachdetach/statusupdater/node_status_updater.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func (nsu *nodeStatusUpdater) UpdateNodeStatuses() error {
 			// If node does not exist, its status cannot be updated.
 			// Do nothing so that there is no retry until node is created.
 			klog.V(2).Infof(
-				"Could not update node status. Failed to find node %q in NodeInformer cache. Error: '%v'",
```
Comment (jsafrane) [inline pkg/controller/volume/attachdetach/statusupdater/node_status_updater.go]: Can you please rephrase so it works without `Error:`? Something like "Could not update node status: %s". `err` usually contains "node XYZ not found" or something like that. IMO we don't need to mention NodeInformer - almost everything has informer in the controller.

In addition, fixing all "Error:" in the attach/detach controller would be great!

Comment (houjun41544) [reply pkg/controller/volume/attachdetach/statusupdater/node_status_updater.go]: Ok,I will fix these later.

File: pkg/controller/nodelifecycle/node_lifecycle_controller.go [hunk 2]
```diff
@@ -677,7 +677,7 @@ func (nc *Controller) monitorNodeHealth() error {
 			}
 			return false, nil
 		}); err != nil {
-			klog.Errorf("Update health of Node '%v' from Controller error: %v. "+
+			klog.Errorf("Update health of Node %s from Controller error: %v. "+
```
Comment (neolit123) [inline pkg/controller/nodelifecycle/node_lifecycle_controller.go]: should be `%q`

Comment (houjun41544) [reply pkg/controller/nodelifecycle/node_lifecycle_controller.go]: All other log infos use %s or %v to format node.name in this file.

**Pull request closed without merging**